docs/releases/1.8.3.txt

```diff
@@ -50,3 +50,6 @@ Bugfixes
 
 * Fixed recording of applied status for squashed (replacement) migrations
   (:ticket:`24628`).
+
+* Fixed :func:`~django.core.urlresolvers.reverse()` not handling nested
```

Comment:
I think this bug has been around forever and thus doesn't qualify for a backport. Did I miss some justification?